automation/test.sh

```diff
@@ -385,7 +385,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ sig-network ]]; then
     export KUBEVIRT_E2E_FOCUS="\\[sig-network\\]"
   elif [[ $TARGET =~ sig-storage ]]; then
-    export KUBEVIRT_E2E_FOCUS="\\[sig-storage\\]|\\[storage-req\\]"
```

Comment:
I see that now we never export `KUBEVIRT_E2E_FOCUS` with `storage-req`, so no lane will run these tests?
Am I missing anything?

Reply:
They are subsets of other sigs, I tried to document this tag's purpose here in a previous PR
https://github.com/kubevirt/kubevirt/blob/4f637f49be97e6a5db5f0309930caf99597c774e/tests/README.md?plain=1#L26-L28